src/Vault.sol

```diff
@@ -67,6 +67,24 @@ contract Vault is ERC4626, ERC20Permit, ILiquidationSource, Ownable {
    */
   event ClaimerSet(Claimer previousClaimer, Claimer newClaimer);
 
+  /**
+   * @notice Emitted when a new LiquidationPair has been set.
+   * @param previousLiquidationPair Address of the previous liquidationPair
+   * @param newLiquidationPair Address of the new liquidationPair
+   */
+  event LiquidationPairSet(
+    LiquidationPair previousLiquidationPair,
```

Comment:
No need to emit the previous liquidation pair; that data is available in the previous event

Reply:
Fixed: https://github.com/pooltogether/v5-vault/pull/6/commits/3888878112483a5f5053c7e5dd3ce7aeb60bcaa3